app/controllers/ContactEmailController.scala

```diff
@@ -53,20 +53,26 @@ class ContactEmailController @Inject() (
         case Some(value) => form.fill(value)
       }
 
-      Ok(view(preparedForm, affinityType, mode))
+      Ok(view(preparedForm, affinityType, getContactName(request.userAnswers, affinityType), mode))
   }
 
+  private def getContactName(userAnswers: UserAnswers, affinityType: AffinityType): String =
+    (userAnswers.get(ContactNamePage), affinityType) match {
+      case (Some(contactName), Organisation) => contactName
+      case _                                 => ""
```

Comment:
There's a message key to provide default text if the name isn't there. It should never happen anyway except when using stub data locally, but probably better to use the default text anyway. The key is `default.firstContact.name` and you'll need to add an implicit messages to the method.

Reply:
This makes sense will change.